### PR TITLE
New italian translations

### PR DESCRIPTION
**Pull request by giuliaghisini**
add new blocks, control panel and other translations for Italian.
Please add these updates to volto core in next release

### CODE REVIEW

File: locales/it/LC_MESSAGES/volto.po [hunk 1]
```diff
@@ -69,19 +69,19 @@ msgstr "Aggiungi un nuovo insieme di campi"
...
 
 #: components/manage/Controlpanels/UsersControlpanel
 msgid "Add new group"
-msgstr ""
+msgstr "Aggiungi un nuovo gruppo"
 
 #: components/manage/Controlpanels/UsersControlpanel
 msgid "Add new user"
-msgstr ""
+msgstr "Nuovo utente"
```
Comment (cekk) [inline locales/it/LC_MESSAGES/volto.po]: Per coerenza, forse va messo "Aggiungi un nuovo utente"?

File: locales/it/LC_MESSAGES/volto.po [hunk 2]
```diff
@@ -776,7 +776,7 @@ msgstr "Nessuna immagine selezionata"
 
 #: components/manage/Blocks/Listing/ListingBody
 msgid "No items found in this container."
-msgstr ""
+msgstr "Nessuno elemento trovato in questo contenitore."
```
Comment (cekk) [inline locales/it/LC_MESSAGES/volto.po]: Nessun?

File: locales/it/LC_MESSAGES/volto.po [hunk 3]
```diff
@@ -1482,11 +1482,11 @@ msgstr "Titolo"
 
 #: config/Blocks
 msgid "toc"
-msgstr ""
+msgstr "Table of contents"
```
Comment (cekk) [inline locales/it/LC_MESSAGES/volto.po]: Non c'è una frase in italiano che ci sta bene?

**Comment by sneridagh** [comment]
@giuliaghisini you should run `yarn i18n` to ensure the new translations are on the .po otherwise Travis will complain.

File: locales/it/LC_MESSAGES/volto.po (hunk 3 above)
Comment (pnicolli) [reply locales/it/LC_MESSAGES/volto.po]: Mi viene in mente solo "Indice dei contenuti"

**Comment by tisto** [comment]
We could improve the Travis output on the i18n check a bit to make this problem a bit more obvious to people.

**Comment by giuliaghisini** [comment]
> @giuliaghisini you should run `yarn i18n` to ensure the new translations are on the .po otherwise Travis will complain.

yess, i do that when i wrote translations. But... i don't know why travis was not happy.. now i run yarn i18n, fixed and add some translation. Now Travis complain and all it's ok

**Comment by sneridagh** [comment]
@giuliaghisini now is green, can you fix conflicts now to be able to merge? Thanks!

**Comment by giuliaghisini** [comment]
> @giuliaghisini now is green, can you fix conflicts now to be able to merge? Thanks!

resolved!

**Pull request merged**